src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationCommentsAfterMethodCall.java

```diff
@@ -0,0 +1,27 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+/* Config: default */
+
+public class InputCommentsIndentationCommentsAfterMethodCall {
+    public static void main() {
+        InputCommentsIndentationCommentsAfterMethodCall.flaMap(4,5)
+           // ok
+           .flaMap()
+           .flaMap();
+        // ok
+    }
+    public static void main1() {
+        InputCommentsIndentationCommentsAfterMethodCall.flaMap(4,5)
+                // ok
+                .flaMap()
+                .flaMap();
+               // violation
+    }
```

Comment:
I added comments in this part which shows the violation that CommentIndentation detects

Reply:
Resolved.